README.md

```diff
@@ -1 +1,31 @@
-# scafeinder
\ No newline at end of file
+# MeanApp
+
+This project was generated with [angular-cli](https://github.com/angular/angular-cli) version 1.0.0-beta.28.3.
```

Comment:
Seriously? beta version? In your package.json version is 1.3.2, be attentive when copypaste